opentelemetry-sdk/src/opentelemetry/sdk/logs/__init__.py

```diff
@@ -244,20 +244,58 @@ def force_flush(self, timeout_millis: int = 30000) -> bool:
         return True
 
 
+# skip natural LogRecord attributes
+# http://docs.python.org/library/logging.html#logrecord-attributes
+_RESERVED_ATTRS = frozenset(
```

Comment:
We are skipping now to separate the user-defined extra attributes but we will revisit this later if needed.